proto/recon.proto

```diff
@@ -112,6 +112,24 @@ message ResolveIdsResponse {
   repeated Entity entities = 1;
 }
 
+message ResolvePlaceNamesRequest {
+  message Place {
+    string name = 1;
+    string type = 2;
+  }
+  repeated Place places = 1;
+}
```

Comment:
Is it worth having a non-repeated version for simple use?

/v1/find/entity?description="capital city in California"
/v1/find/entity?description="santa clara"&type=County